.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,7 @@
+-   id: check-python-versions
+    name: check-python-versions
+    description: This is a tool for Python package maintainers who want to explicitly state which Python versions they support.
+    entry: check-python-versions
+    language: python
+    pass_filenames: false
+    always_run: true
```

Comment:
git shows a whitespace error (missing newline at EOF)

Reply:
This was intentionally left out to match the YAML style in `check-manifest` :-)

Reply:
Huh?  I don't see any git whitespace errors in https://github.com/mgedmin/check-manifest/pull/102/files

Reply:
Hm... GitHub does not show the emtpy line....

https://github.com/mgedmin/check-manifest/blob/master/.pre-commit-hooks.yaml

Reply:
What empty line?  There's no empty line.

I think we're seeing a mismatch between your text editor of choice and the rest of the world.  On Unix every line, including the last line in a file, is supposed to end in a line terminator (\n).  Some other operating systems (DOS/Windows) use a line _separator_ instead of a line _terminator_, so the last line is not followed by anything.

I think Emacs is one text editor that flaunts Unix conventions and shows a phantom empty line at the bottom of every correctly terminated Unix file.  I know that Vim doesn't (and Vim goes out of its way to add the \n after the last line in a file, unless you explicitly set a bunch of options to prevent it).

git tries to enforce Unix conventions and so git diff shows a warning if you try to add a file without the trailing newline.  GitHub copies this.